compiler/rustc_hir/src/lang_items.rs

```diff
@@ -343,4 +343,7 @@ language_item_table! {
     Range,                   sym::Range,               range_struct,               Target::Struct;
     RangeToInclusive,        sym::RangeToInclusive,    range_to_inclusive_struct,  Target::Struct;
     RangeTo,                 sym::RangeTo,             range_to_struct,            Target::Struct;
+    Send,                    sym::send,                send_trait,                 Target::Trait;
+    UnwindSafe,              sym::unwind_safe,         unwind_safe_trait,          Target::Trait;
+    RefUnwindSafe,           sym::ref_unwind_safe,     ref_unwind_safe_trait,      Target::Trait;
```

Comment:
I've just seen in https://github.com/rust-lang/rust/pull/86603 that `Send` is made a lang item again.

I don't see why this PR needs to add new lang items, all of these could be diagnostic items instead.

Reply:
That's true, I forgot that diagnostic items were a thing.

Reply:
@petrochenkov Thanks for pointing this out, I'll try to have a fix done in the next few days.

Reply:
@petrochenkov The fix is up for review https://github.com/rust-lang/rust/pull/86726